src/main/java/com/vinylteam/vinyl/service/impl/DefaultUniqueVinylService.java

```diff
@@ -1,5 +1,6 @@
 package com.vinylteam.vinyl.service.impl;
```

Comment:
Can we remove "impl" package ?

Reply:
Do you mean to put all service implementations and interfaces into one package com.vinylteam.vinyl.service?